src/parser/jobs/dnc/modules/Procs.tsx

```diff
@@ -1,37 +1,41 @@
 import {Trans} from '@lingui/react'
-import {ActionLink} from 'components/ui/DbLink'
+import {DataLink} from 'components/ui/DbLink'
 import {Procs as CoreProcs} from 'parser/core/modules/Procs'
 import React from 'react'
 
 export class Procs extends CoreProcs {
 	override trackedProcs = [
 		{
-			procStatus: this.data.statuses.FLOURISHING_FAN_DANCE,
+			procStatus: this.data.statuses.THREEFOLD_FAN_DANCE,
 			consumeActions: [this.data.actions.FAN_DANCE_III],
 		},
 		{
-			procStatus: this.data.statuses.FLOURISHING_CASCADE,
-			consumeActions: [this.data.actions.REVERSE_CASCADE],
+			procStatus: this.data.statuses.FOURFOLD_FAN_DANCE,
+			consumeActions: [this.data.actions.FAN_DANCE_IV],
 		},
 		{
-			procStatus: this.data.statuses.FLOURISHING_FOUNTAIN,
-			consumeActions: [this.data.actions.FOUNTAINFALL],
+			procStatus: this.data.statuses.FLOURISHING_SYMMETRY,
+			consumeActions: [this.data.actions.REVERSE_CASCADE, this.data.actions.RISING_WINDMILL],
 		},
 		{
-			procStatus: this.data.statuses.FLOURISHING_SHOWER,
-			consumeActions: [this.data.actions.BLOODSHOWER],
+			procStatus: this.data.statuses.FLOURISHING_FLOW,
+			consumeActions: [this.data.actions.FOUNTAINFALL, this.data.actions.BLOODSHOWER],
 		},
 		{
-			procStatus: this.data.statuses.FLOURISHING_WINDMILL,
-			consumeActions: [this.data.actions.RISING_WINDMILL],
+			procStatus: this.data.statuses.FLOURISHING_STARFALL,
+			consumeActions: [this.data.actions.STARFALL_DANCE],
+		},
+		{
+			procStatus: this.data.statuses.FLOURISHING_FINISH,
+			consumeActions: [this.data.actions.TILLANA],
 		},
 	]
 
 	override showDroppedProcSuggestion = true
 	override droppedProcIcon = this.data.actions.FOUNTAINFALL.icon
 	override droppedProcContent =
 		<Trans id="dnc.procs.suggestions.drops.content">
-			Avoid dropping your procs unless absolutely necessary. If you have to drop one to keep your Esprit from overcapping, <ActionLink {...this.data.actions.RISING_WINDMILL} /> or <ActionLink {...this.data.actions.REVERSE_CASCADE} /> will lose the least DPS overall.
+			Avoid dropping your procs unless absolutely necessary. If you have to drop one to keep your Esprit from overcapping, <DataLink action="RISING_WINDMILL" /> or <DataLink action="REVERSE_CASCADE" /> will lose the least DPS overall.
```

Comment:
With the consolidation to two shared procs, should we update this copy to use the name of the shared proc instead of the actions that can cast it?  Will we even still need this module, since proc duration should more than cover using up two procs?

Reply:
Yeah, we can change the copy. I'd rather leave the suggestion present because while it's significantly less likely now, it's still technically possible to wind up in a situation where Tech finish's Esprit spam makes it difficult to get all the procs in. I'll re-evaluate the necessity of the suggestion after we actually get our hands on the post-expac build though.